app/portainer/home/EnvironmentList/KubeconfigSelection.tsx

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+
+import { EnvironmentId } from '@/portainer/environments/types';
+
+export function useSelection() {
```

Comment:
we also name the file as the main function or component. this file should be `useSelection.ts`